carbon-design-system/devtools
# Migrate to manifest v3

Closes #84 

Add support for Manifest v3

##### New

- Add new package for Manifest v2 (FIrefox issue)

##### Changed

- Update functions in `utilities` package to support v3
- Update main package to support v3

#### Testing / Reviewing

- Navigate to `web-extenstion` (Manifest v3) or `web-extension-v2` (Manifest v2) package and run `yarn watch` or `yarn build`.
- Setup Chrome (v3) / Firefox (v2) browser with custom extension. 
- Open any website built using Carbon.
- Verify Devtools functionality.
